src/include/sof/audio/pcm_converter.h

```diff
@@ -14,6 +14,7 @@
 #ifndef __SOF_AUDIO_PCM_CONVERTER_H__
 #define __SOF_AUDIO_PCM_CONVERTER_H__
 
+#include <config.h>
```

Comment:
Includes should be sorted by increasing visibility, config is more global than ipc/* stuff, so it should be after it. 
config.h can only be included earlier if you have to decide what-to-include depending on CONFIG_ symbols